### PR TITLE
Feature/#33 implement add 03 events   01 events sample

### LINKED ISSUE #33: Add 03 Events - 01 Events sample

### PR DESCRIPTION
**Pull request by nasdan**
Close #33

### CODE REVIEW

File: 03 Events/01 Events/README.md [hunk 1]
```diff
@@ -31,7 +33,147 @@ Summary steps:
...
+const dataRetriever = (countTries) => {
+  const emitter = new EventEmitter();
+  deferredProcess(countTries, emitter);
+
+  return emitter;
+};
+
+```
+
+- For example, we are going to retrieve `10 countries`:
```
Comment (JaimeSalas) [inline 03 Events/01 Events/README.md:99]: They are tries not countTries

Comment (nasdan) [reply 03 Events/01 Events/README.md:99]: Yes, I change it on next sample. I'm going to update this one too

Comment (JaimeSalas) [reply 03 Events/01 Events/README.md:99]: OK

Comment (nasdan) [reply 03 Events/01 Events/README.md:99]: Done

**Review by JaimeSalas — APPROVED** [review]

**Pull request merged**